.github/steps/3-step.md

```diff
@@ -3,15 +3,12 @@
 Great work doing that research and finding a potential collaboration opportunity.
 Not only did we find some new ideas to help organize extracurricular activities, but we did all that quickly too.
 
-Plenty of time to focus on the fun stuff, like teaching our awesome students! 🌱
```

Comment:
I feel like sometimes the story goes too far into the cringe side of things. Maybe it's just me...